### PR TITLE
CLOUDP-115902: [mcli] Allow to update feature controls with a file

### PR DESCRIPTION
**Pull request by gssbzn**
## Proposed changes
_Jira ticket:_ CLOUDP-115902

## Checklist
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added any necessary documentation in document requirements section listed in [CONTRIBUTING.md](https://github.com/mongodb/mongocli/blob/master/CONTRIBUTING.md) (if appropriate)
- [x] I have addressed the @mongodb/docs-cloud-team comments (if appropriate)
- [x] I have updated [e2e/E2E-TESTS.md](https://github.com/mongodb/mongocli/blob/master/e2e/E2E-TESTS.md) (if an e2e test has been added)
- [x] I have run `make fmt` and formatted my code

### CODE REVIEW

File: docs/mongocli/command/mongocli-cloud-manager-featurePolicies-update.txt [hunk 1]
```diff
@@ -32,13 +34,17 @@ Options
      - Type
      - Required
      - Description
+   * - -f, --file
+     - string
+     - false
+     - File name to use, optional file with a json cluster configuration.
```
Comment (andreaangiolillo) [inline docs/mongocli/command/mongocli-cloud-manager-featurePolicies-update.txt:40]:
```suggestion
     - File name to use, optional file with a json cluster configuration.
```

Is this usage correct?

**Review by andreaangiolillo — APPROVED** [review]
LGTM

File: docs/mongocli/command/mongocli-cloud-manager-featurePolicies-update.txt (hunk 1 above)
Comment (gssbzn) [reply docs/mongocli/command/mongocli-cloud-manager-featurePolicies-update.txt:40]: no, copy-pasta 🙃

**Review by andreaangiolillo — APPROVED** [review]
LGTM

**Review by sarahsimpers — APPROVED** [review]
LGTM

**Pull request merged**